x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/use_edit_transform_flyout.ts

```diff
@@ -154,10 +191,13 @@ const isFormValid = (fieldsState: EditTransformFlyoutFieldsState) =>
 // Updates a form field with its new value,
 // runs validation and populates
 // `errorMessages` if any errors occur.
-const formFieldReducer = (state: Field, value: string): Field => {
+const formFieldReducer = (state: FormField, value: string): FormField => {
   return {
     ...state,
-    errorMessages: state.isOptional && value.length === 0 ? [] : validate[state.validator](value),
+    errorMessages:
```

Comment:
If the update fails, a toast notification is correctly displayed. But when I dismiss the toast, the flyout closes, not allowing the user to correct any mistakes. The flyout should remain open after the toast error is dismissed.

Reply:
Good find, will be solved in a follow-up.